packages/compiler/src/plugins/typescript/visitor.ts

```diff
@@ -199,56 +199,61 @@ export class Visitor {
+
+        const blockTags = comment.blockTags.filter((tag) => tag.tag !== "@default" && tag.tag !== "@deprecated");
+        if (blockTags.length > 0) {
+            documentation += "\n\n";
+            documentation += blockTags.map((tag) => `${tag.tag} ${tag.content}`).join("\n");
         }
+
         return this.compiler.renderBlock(documentation);
     }
 }
 
-function getCommentTag(comment: Comment | undefined, tagName: string) {
-    if (comment == null || comment.tags == null) {
-        return undefined;
-    }
-    return comment.tags.filter((tag) => tag.tagName === tagName)[0];
+function getCommentTagValue(comment: Comment | undefined, tagName: string) {
+    const maybeTag = comment?.getTag(`@${tagName}`);
+    return maybeTag?.content.map((part) => part.text.trim()).join("\n");
 }
 
 function getDefaultValue(ref: ParameterReflection | DeclarationReflection): string | undefined {
-    if (ref.defaultValue != null) {
-        return ref.defaultValue;
+    // N.B. TypeDoc no longer sets defaultValue for enum members as of v0.23, see https://typedoc.org/guides/changelog/#v0.23.0-(2022-06-26)
+    // Also, we typically expect enum member values to only have literal types, so we can just use the type value.
+    if (ref.kind === ReflectionKind.EnumMember && ref.type?.type === "literal") {
+        return ref.type?.value?.toString();
     }
-    const defaultValue = getCommentTag(ref.comment, "default");
-    if (defaultValue !== undefined) {
-        return defaultValue.text.trim();
+
+    return ref.defaultValue ?? getCommentTagValue(ref.comment, "default");
+}
+
+function getSourceFileName(reflection: Reflection): string | undefined {
+    if (reflection.isDeclaration() || isSignatureReflection(reflection)) {
+        if (reflection.sources !== undefined) {
+            const { fullFileName } = reflection.sources[0];
+            // fullFileName relative to cwd, so it can be saved in a snapshot (machine-independent)
+            return fullFileName && relative(process.cwd(), fullFileName);
+        }
     }
     return undefined;
 }
 
-function getSourceFileName({ sources = [] }: Reflection): string | undefined {
-    const source = sources[0];
-    const fileName = source && source.file && source.file.fullFileName;
-    // filename relative to cwd, so it can be saved in a snapshot (machine-independent)
-    return fileName && relative(process.cwd(), fileName);
+function isSignatureReflection(reflection: Reflection): reflection is SignatureReflection {
+    return reflection.variant === "signature";
 }
 
-function getSourceUrl({ sources = [] }: Reflection): string | undefined {
-    const source = sources[0];
-    return source && source.url;
+function getSourceUrl(reflection: Reflection): string | undefined {
+    if (reflection.isDeclaration() || isSignatureReflection(reflection)) {
+        if (reflection.sources !== undefined) {
+            const source = reflection.sources[0];
+            return source && source.url;
```

Comment:
could/should we do this now?
```suggestion
            return source?.url;
```